test/betting/helpers/bootstrap.jsligo

```diff
@@ -34,13 +34,12 @@ export const bootstrap = () => {
   };
 
   // Boostrapping BETTING contract
-  let betting_path = "../../../src/contracts/jsligo/betting/main.jsligo";
-  let iBis = Test.run( (x : Types.storage) => x, init_storage);
```

Comment:
This `Test.run` call is missing after your changes

Reply:
yes, we now perform the run from inside the originate_from_file function